doc/man3/OSSL_STORE_INFO.pod

```diff
@@ -122,13 +122,12 @@ given to L<OSSL_STORE_open_file(3)>, the returned name will be the path of
 each object, so if C</foo/bar> was given and that path has the file
 C<cookie.pem>, the name C</foo/bar/cookie.pem> will be returned.
 
-The returned URI is considered canonical and must be unique and permanent
-for the storage where the object (or collection of objects if that's what
-each name represents) resides.
-It is up to each loader to ensure this.
```

Comment:
I had intended to only remove the 'It is up to..." sentence and leave the previous one, but now that I look at your change, the previous sentence is not really needed, either, except that it was the only place where "canonical" was listed as a requirement.